packages/core/src/util/jwt.ts

```diff
@@ -101,29 +110,31 @@ export async function verifyJwt(
   options = { ...defaultVerifyJwtOptions, ...options };
 
   const creds = getXsuaaServiceCredentials(token);
+  const jku = getVerificationKeyURL(token);
```

Comment:
Minor: `jku` might not be that helpful of a name when coming to this code without much knowledge of jwts. [According the RFC](https://tools.ietf.org/html/rfc7515#section-4.1.2), `jku` stands for "JWK Set URL" and JWK stands for JSON web key... Not really a fancy name in there, either, but maybe you can think of something.

Reply:
I used verificationKeyURL which was the suggestion from the IDE due to the getter function and is better then jku I would say.